tests/Doctrine/Tests/ORM/Functional/PaginationTest.php

```diff
@@ -179,25 +336,44 @@ public function testCountQueryStripsParametersInSelect()
         // if select part of query is replaced with count(...) paginator should remove
         // parameters from query object not used in new query.
         $this->assertCount(1, $getCountQuery->invoke($paginator)->getParameters());
-        $this->assertCount(3, $paginator);
+        $this->assertCount(9, $paginator);
     }
 
     public function populate()
     {
-        for ($i = 0; $i < 3; $i++) {
+        $groups = [];
```

Comment:
hmm, given the big refactoring done to fix the pagination for PostgreSQL and SQL server, this will probably not be backported though, so it may not be worth it)